config/routes.rb

```diff
@@ -1,3 +1,13 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'sessions/new'
+  root 'microposts#index'
+  get '/signup', to: 'users#new'
```

Comment:
Let clear up our routes, i don't think really need line 4 and 5 right, according to the project instructions, the below sessions are ok for the project; 
get '/login', to: 'sessions#new'
post '/login', to: 'sessions#create'
delete '/logout', to: 'sessions#destroy'